pages/storefront/edit.tsx

```diff
@@ -99,7 +101,7 @@ export default function Edit() {
 
   const values = reduceFieldData(fields);
 
-  const subdomainUniqueness = validateSubdomainUniqueness(ar, wallet.pubkey);
+  const subdomainUniqueness = validateSubdomainUniqueness(ar, publicKey?.toString());
```

Comment:
We probably want to check there's a pubKey before reaching this point, but I'm not pretty sure